Makefile.org

```diff
@@ -685,7 +685,7 @@ install_docs:
 		(cd `$(PERL) util/dirname.pl $$i`; \
 		pod2man \
 			--section=$$sec --center=OpenSSL \
-			--release=$(VERSION) `basename $$i`) \
+			--release="OpenSSL $(VERSION)" `basename $$i`) \
```

Comment:
Seems to depend.  Just for kicks, I just did:

`man man` - version number
`man perl` - "perl" followed by version number
`man 2 connect` - simply "Linux"

So I'd say people seem not to care very much...  Me, I would prefer that reverted.